brutils/cep.py

```diff
@@ -0,0 +1,25 @@
+from random import randint
+
+
+# FORMATTING
+############
+
+
+# OPERATIONS
+############
+
+
+def is_valid(cep):  # type: (str) -> bool
+    """
+    Returns when CEP is valid, input should be a digit
+    string of proper length. Doesn't validate if it's real,
+    'cause only the "Correios" Base does.
```

Comment:
Esse disclaimer podia estar na documentação (READMEs) também. O quê achas? Assim a pessoa que for utilizar já fica ciente de como a função funciona.